tests/integration/managed-array-tracked-test.js

```diff
@@ -134,6 +138,37 @@ if (CUSTOM_MODEL_CLASS) {
       );
     });
 
+    // We have run into scenarios like these with users stashing M3 Arrays on POJOs in services, and then accessing them and
+    // modifying during a render
+    test('Can modify a managed array after creation without triggering rerendering assertions', async function (assert) {
```

Comment:
I'm assuming this test would fail when the ember infra throws an error ?

Reply:
Yup, Ember would error out